lib/scripts/default.nix

```
@@ -1,7 +1,7 @@
 { pkgs ? import <nixpkgs> {} }:
 
 let
-  py2hwsw_commit = "f24f15cfa8b21f660b24a15af57423d0c501e041"; # Replace with the desired commit.
+  py2hwsw_commit = "42ec36ca141ef7aeb0ba0bcfe0af5d8344c029fc"; # Replace with the desired commit.
```

I have added a new [PR in py2hwsw](https://github.com/IObundle/py2hwsw/pull/24) that contains dependencies for the current iob-soc.
Updating the py2hwsw to that commit should fix the ghactions tests.
```suggestion
  py2hwsw_commit = "7d3f852889ebeb986937985d2d80215f6aeccf10"; # Replace with the desired commit.
```